doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -589,60 +589,54 @@ Shown below is the far-field energy-density profile around the focal length for
 
-2. Each $m$-simulation in the Fourier-series expansion is independent of the others. The simulations can therefore be executed simultaneously using an embarassingly parallel approach.
+2. Each $m$-simulation in the Fourier-series expansion is independent of the others. The simulations can therefore be executed simultaneously using an [embarassingly parallel](https://meep.readthedocs.io/en/latest/Parallel_Meep/#different-forms-of-parallelization) approach.
 
 ![](../images/cyl_nonaxisymmetric_source_flux_vs_m.png#center)
 
-The Poynting flux computed in cylindrical coordinates can be converted to 3d Cartesian coordinates with the use of various scaling factors:
+As a demonstration, we compute the [extraction efficiency of an LED](https://meep.readthedocs.io/en/latest/Python_Tutorials/Local_Density_of_States/#extraction-efficiency-of-a-light-emitting-diode-led) from a point dipole at $r = 0$ and three different locations at $r > 0$. These results are compared to an [identical calculation in 3d](https://github.com/NanoComp/meep/blob/1fe38999997f1825054fc978e473327c77169671/python/examples/extraction_eff_ldos.py#L100-L187) for which the extraction efficiency is 0.333718. Results for the simulations in cylindrical coordinates are shown in the table below. At this resolution, the relative error is at most ~4%. The error decreases with increasing resolution.
 
-- To convert the radiated flux from a point dipole at $r=0$ in cylindrical coordinates to 3d, the result must be multiplied by $2 / (\pi \Delta r)^2$ where $\Delta r$ is the grid size ($1/resolution$).
+| `rpos` | **extraction efficiency** | **relative error** |  `M` (number of `m`-simulations) |
+|:------:|:-------------------------:|:------------------:|:--------------------------------:|
+|    0   |          0.310405         |        0.042       |                 1                |
+|   3.5  |          0.299407         |        0.041       |                56                |
+|   6.7  |          0.300334         |        0.036       |                101               |
+|   9.5  |          0.302234         |        0.028       |                141               |
```

Comment:
Oof that's a lot of `m` simulations...

Reply:
In this example, I intentionally used a large value (`9.5`) for the source position in $r$ (which requires a good number of individual simulations as you have noted) simply to demonstrate convincingly that the approach involving a Fourier-series expansion works even when many `m`-simulations are involved. For most practical applications, it is probably unlikely that such a large value would be used.

Also, note that it is likely that I could have probably used fewer simulations without sacrificing accuracy in the converged results. The threshold flux value used to determine when to truncate the expansion is chosen somewhat arbitrarily as `flux_tol = 1e-5` in this tutorial. We can try increasing this value.